newerhoods/server.R

```diff
@@ -230,9 +241,7 @@ function(input, output) {
 
 
   ##### Interactive Map #####
-
-  output$map <-
-    renderLeaflet({
+  map_reactive <- reactive({
```

Comment:
To make the snapshot feature work, I had to make the leaflet map it's own reactive variable. This seems to not effect anything else but @kaushik12 let me know if you think there is a better way of doing this.